lib/galaxy/web/framework/webapp.py

```diff
@@ -643,6 +643,45 @@ def user_checks(self, user):
         """
         self.check_user_library_import_dir(user)
 
+    def associate_user_history(self, user, prev_galaxy_session=None):
+        # Associated the current user's last accessed history (if exists) with their new session
```

Comment:
s/Associated/Associate/

Also, you can use the `"""`...`"""` syntax for proper method documentation.

Reply:
Fixed nine other typos while browsing through, but missed the one here in the comment text I actually copied, of course :)

Reply:
Nice, thanks!